index.html

```diff
@@ -178,31 +179,6 @@
           break;
       }
     }
-
-    function calculateChartsSize() {
```

Comment:
I suggest for us to use some kind of a code convention: Place the function definitions in the bottom so we can easily see the program flow without getting into details.